py/defs.bzl

```diff
@@ -104,7 +91,16 @@ def _py_binary_or_test(name, rule, srcs, main, data = [], deps = [], resolutions
         target_compatible_with = kwargs.get("target_compatible_with", []),
     )
 
-def py_binary(name, srcs = [], main = None, **kwargs):
+def _normalize_label(src):
+    if src.startswith("//"):
+        return Label(src)
+    else:
+        return Label("//%s:%s" % (native.package_name(), src.lstrip(":")))
+
+def _normalize_srcs(srcs):
+    return [_normalize_label(it) for it in srcs]
```

Comment:
@dzbarsky is there a better way here? The motivation is that if the user gives us `:main.py`, we can't immediately tell without normalization that `main.py` is already on the list. Applies to labels generally; consider absolute label to main.